r2dbc-mysql/src/main/java/JasyncInsertFakeRow.kt

```diff
@@ -0,0 +1,50 @@
+package com.github.jasync.r2dbc.mysql
+
+import io.r2dbc.spi.Row
+
+/**
+ * A fake row considers last inserted ID for support [JasyncStatement.returnGeneratedValues].
+ *
+ * @see JasyncResult
+ * @see JasyncStatement
+ * @see JasyncInsertFakeMetadata
+ * @see io.r2dbc.spi.Statement
+ */
+internal class JasyncInsertFakeRow(private val generatedKeyName: String, private val lastInsertId: Long) : Row {
```

Comment:
Same naming issue here.

Reply:
Naming it `JasyncInsertSyntheticRow` now.